README.md

```diff
@@ -49,8 +65,32 @@ You must to issue personal access token in advance.
 $ COMPOSER_PROCESS_TIMEOUT=0 composer run
 ```
 
+If you use Docker, alternatively run below command.
+
+```bash
+$ docker-compose up # It take a while to install composer & npm libraries in first 
+```
+
 And go to http://localhost:8888/index_dev.php/post/:post_number
 
+### Runnin in production
+
+You can run production Apache server with [DockerHub Automated Build Image](https://hub.docker.com/r/kokuyouwind/esaba/).
```

Comment:
> DockerHub Automated Build Image

it is good to use docker image name, i.e. `kokuyouwind/esaba:latest`.

Reply:
fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880